ruby/ql/lib/codeql/ruby/frameworks/ActionController.qll

```diff
@@ -119,13 +121,6 @@ private class ActionControllerContextCall extends MethodCall {
   ActionControllerControllerClass getControllerClass() { result = controllerClass }
 }
 
-/**
- * A call to the `params` method to fetch the request parameters.
- */
-abstract class ParamsCall extends MethodCall {
```

Comment:
Does moving this (and `CookiesCall`) technically break the deprecation policy as these are no longer exported from this library? Wondering if we should re-export these to avoid potential problems if a query happens to rely on `ActionController.qll` exporting these classes.

Reply:
That's a good point. I confirmed it won't cause any problems with the internal query repo, but let me see if I can ex-export them.

Reply:
It seems that you can't just export a class as an alias under the same name without getting errors about conflicting definitions in files that import both libraries. So I think we have three options:

1. Leave it as is and break the deprecation policy.
2. Rename the classes that got moved to `Rails.qll` and export the old names as aliases in `ActionView.qll` and `ActionController.qll`.
3. Move them back to `ActionView.qll` and `ActionController.qll`.

Reply:
OK, I think I've found a nicer solution than any of those. The classes in `Rails.qll` can go inside a module named `Rails`, and then we can introduce deprecated aliases in `ActionView.qll` and `ActionController.qll` without causing conflicting definitions anywhere.

Reply:
Nice, I find it a bit neater having these enclosed in a separate module anyway, so this seems like a good solution.